### PR TITLE
fix a bug in FVM kernel

### PR DESCRIPTION
**Pull request by Zhengyu-Huang**
### Description

In the FVM kernel, 
When increment is off, and for periodic top/bottom case: 

the top element tendency is updated in 2 places, in the first place(ie = 1) it is set as = the second place(ie=nelem) should be updated as +=

- [ ] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [ ] Unit tests are included OR N/A.
- [ ] Code is exercised in an integration test OR N/A.
- [ ] Documentation has been added/updated OR N/A.

### CODE REVIEW

**Comment by Zhengyu-Huang** [comment]
bors try

File: src/Numerics/DGMethods/DGFVModel_kernels.jl [hunk 1]
```diff
@@ -590,7 +590,7 @@ A finite volume reconstruction is used to construction `Fⁱⁿᵛ⋆`
                             else
                                 T = local_tendency[s]
                             end
-                            tendency[n, s, eH + eV_up] = T
+                            tendency[n, s, eH + eV_up] += T
```
Comment (mwarusz) [inline src/Numerics/DGMethods/DGFVModel_kernels.jl:593]: Good catch. Does this mean that we also shouldn't scale by beta again here ? So the whole thing can be simplified to just `tendency += local_tendency` regardless of `increment` ?

Comment (Zhengyu-Huang) [reply src/Numerics/DGMethods/DGFVModel_kernels.jl]: What is beta? modify to tendency += local_tendency

Comment (jkozdon) [reply src/Numerics/DGMethods/DGFVModel_kernels.jl]: `beta` is a scaling factor for the previous tendency. See the description here:
https://github.com/CliMA/ClimateMachine.jl/blob/c693e9e7d85ceff7ba55f71ebdf3ee577d36dc71/src/Numerics/DGMethods/DGModel.jl#L189-L202

**Review by jkozdon — COMMENTED** [review]
Can we add a test to make sure this gets hit? Must not have before I guess?

File: src/Numerics/DGMethods/DGFVModel_kernels.jl [hunk 2]
```diff
@@ -583,14 +583,8 @@ A finite volume reconstruction is used to construction `Fⁱⁿᵛ⋆`
...
-                            if β != 0
-                                T =
-                                    local_tendency[s] +
-                                    β * tendency[n, s, eH + eV_up]
-                            else
-                                T = local_tendency[s]
-                            end
-                            tendency[n, s, eH + eV_up] = T
+                            # This tendency has already been updated in the first element (eV_up=1)
+                            tendency[n, s, eH + eV_up] += local_tendency[s]
```
Comment (jkozdon) [inline src/Numerics/DGMethods/DGFVModel_kernels.jl:587]:
```julia
                            # This tendency has already been updated in the first element (eV_up=1)
                            tendency[n, s, eH + eV_up] += local_tendency[s]
```

**Pull request closed without merging**